### ISSUE TITLE
Terra i18n generated translation files cause script errors

### ISSUE THREAD

**Issue by kushanson** [cerner/terra-core#914]
# Issue Description
The js files generated in the aggregated-translations folder use a mix of ES6 'import' statements and module.exports. 

## Issue Type
- [x] Bug

## Expected Behavior
the plugin should generate code that uses es6 imports with es6 exports. The code should be generated as follows:
```javascript
import { addLocaleData } from 'react-intl';
import localeData from 'react-intl/locale-data/en';

addLocaleData(localeData);

// aggregated messages ...

let i18nData = {
  areTranslationsLoaded: true,
  locale: 'en',
  messages,
};

export default i18nData;
```

## Current Behavior
ES6 imports are used with module.exports in the generated code. This may have been acceptable on older versions of webpack. It is currently causing a script error. The generated code looks as follows:
```javascript
import { addLocaleData } from 'react-intl';
import localeData from 'react-intl/locale-data/en';

addLocaleData(localeData);

// messages

module.exports = {
  areTranslationsLoaded: true,
  locale: 'en-GB',
  messages,
};
```

## Steps to Reproduce

1. install terra i18n plugin
2. Use the plugin to generate aggregated-translations
3. Examine the generated files.
4. 

## Environment

* Component Version: "terra-i18n": "^1.6.0",
    "terra-i18n-plugin": "^1.2.0",
    "webpack": "3.6.0",

* Browser Name and  @Version: Chrome Version 61.0.3163.100 (Official Build) (32-bit)
* Operating System and version (desktop or mobile): Desktop Windows 8.1

**Comment by mhemesath** [comment]
What is the script error?

**Comment by mhemesath** [comment]
Also what is your Babel config

**Comment by mhemesath** [comment]
Webpack can read ES6 modules now but that means import and module.exports can’t be mixed as they currently are above. The quick fix is to change the exports from module.exports to simply export{} in the i18n plugin

**Comment by kushanson** [comment]
The script error is: 
Unhandled Rejection (TypeError): Cannot assign to read only property 'exports' of object '#<Object>'

.babelrc:
{
  "presets": [
    ["latest", { "modules": false }],
    "es2015",
    "react"
  ]
}

I like your quick fix since I haven't discovered what babel config makes this work. I'll keep digging into that on my end.

**Comment by kushanson** [comment]
I looked further. It looks like my options are to downgrade webpack to 2.2.0 or wait for your fix to get merged. I'm going to wait for the fix since I can work around the error for now.

Here is the source of info about the breaking change in webpack:
https://github.com/webpack/webpack/issues/4039

**Comment by mhemesath** [comment]
The other option is to compile modules  in your .babelrc file.

**Comment by bjankord** [comment]
Resolved in #916
